### PR TITLE
Make cct run for cloud 5 and 6

### PR DESCRIPTION
**Pull request by vmoravec**
Not tested yet, waiting for admin node to get sle12.

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3273,14 +3280,21 @@ function onadmin_run_cct()
     # - install cct dependencies
     ensure_packages_installed git-core gcc make ruby2.1-devel
 
-    if iscloudver 6plus && [[ -n $cct_tests ]]; then
+    if [ -n $cct_tests ]; then
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: you still need to check for iscloudver 5plus (we're also gating for cloud 4).

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -3273,14 +3280,21 @@ function onadmin_run_cct()
...
         mkdir -p /root/github.com/SUSE-Cloud
         pushd /root/github.com/SUSE-Cloud/
         git clone https://github.com/SUSE-Cloud/cct.git
-        # TODO checkout branches for older versions
         cd cct
+
+        # checkout branches if needed
+        case "$cloudsource" in
+            develcloud5|GM5|GM5+up)
+                git checkout "cloud5"
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: please note you can also pass the branchname to "git clone" directly with -b <branch>.

**Comment by vmoravec** [comment]
Added branch to be checked out on cloning and testcondition `iscloudver 5plus`

**Comment by dirkmueller** [comment]
LGTM, and testing passed.

**Pull request merged**